packages/@aws-cdk/core/test/custom-resource-provider-fixture/index.ts

```diff
@@ -0,0 +1,10 @@
+// tslint:disable: no-console
```

Comment:
Apply the same folder structure to tests as well, please

Reply:
done